lib/curl_setup.h

```diff
@@ -641,7 +641,7 @@ int netware_init(void);
 #endif
 
 /* Single point where USE_NTLM definition might be defined */
-#ifndef CURL_DISABLE_CRYPTO_AUTH
+#if !defined(CURL_DISABLE_NTLM) && !defined(CURL_DISABLE_CRYPTO_AUTH)
```

Comment:
This should probably also apply for the `USE_NTLM` below. So for SSPI, we wouldn't want NTLM either with `CURL_DISABLE_NTLM`.

Reply:
The scope of this if ends at line 662 so USE_NTLM will be disabled even when USE_WINDOWS_SSPI is enabled.
I can try to improve the readability here by adding new lines here and there.

Thanks, your comment led me to the bug in CMake build in this patch which has the exact same error.